pkg/github/github.go

```diff
@@ -44,13 +54,23 @@ func GenerateNewRepositoryName(displayName, namespace, clusterName string) strin
 func GenerateNewRepository(client *github.Client, ctx context.Context, orgName string, repoName string, description string) (string, error) {
 	isPrivate := false
 	appStudioAppDataURL := "https://github.com/" + orgName + "/"
-
+	metrics.GitOpsRepoCreationTotalReqs.Inc()
 	r := &github.Repository{Name: &repoName, Private: &isPrivate, Description: &description}
-	_, _, err := client.Repositories.Create(ctx, orgName, r)
+	_, resp, err := client.Repositories.Create(ctx, orgName, r)
+
+	if 500 <= resp.StatusCode && resp.StatusCode <= 599 {
```

Comment:
What about certain non 5xx response codes? E.g. 429, 403?

Reply:
I just made a generalization that all 5xx errors are server side errors that users cannot fix on their own.  429 is related to rate limiting which we should have in place and 403 could be an unauthorized request (user problem that should not factor into the SLI).  

Let me know if there's a better way to distinguish between user and system errors.